src/lib/queue.js

```diff
@@ -20,6 +21,9 @@ export default function(opts) {
             events.emit('job:run:init', {job});
 
             _id = job._id;
+
+            await apiClient.setState(_id, {state: 'RUNNING', error: undefined});
```

Comment:
I actually intended to enable re-running jobs from `admin`. The idea is if for some reason a schedule change (or a batch of them) failed, ops team would be able to just re-run them from the UI.

Should I still remove this?